components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpoint.java

```diff
@@ -65,50 +68,84 @@ public String jwks() {
 
         try (FileInputStream file = new FileInputStream(keystorePath)) {
             int tenantId = IdentityTenantUtil.getTenantId(tenantDomain);
-            RSAPublicKey publicKey;
+            KeyStore keystore;
+            HashMap<String, Certificate> certKeyPair = new HashMap<>();
```

Comment:
does this name explain what is in the map?

Reply:
I changed the name